packages/frontend/src/lib/sets.ts

```diff
@@ -13,6 +13,7 @@ const SETS = [
   'medium-nft-trader',
 
   'nouns-forker',
+  'protocol-guild',
```

Comment:
I'd push the order up to the top of the list

Reply:
moved to top